pkg/tenant/db/keys.go

```diff
@@ -68,6 +68,43 @@ func (k Key) ObjectID() (id ulid.ULID, err error) {
 	return id, err
 }
 
+// String returns a string representation of the key with the parent ID concatenated
+// with the object ID.
+func (k Key) String() (_ string, err error) {
+	if len(k) != 32 {
+		return "", ErrKeyWrongSize
+	}
+
+	var parentID, objectID ulid.ULID
+	if parentID, err = k.ParentID(); err != nil {
+		return "", err
+	}
+
+	if objectID, err = k.ObjectID(); err != nil {
+		return "", err
+	}
+
+	return parentID.String() + objectID.String(), nil
```

Comment:
It might be useful to put some separator between the parent and object ID for readability; e.g. a `-` or a `/`?